schedule/yast/raid/testdata/lvm_raid1.yaml

```diff
@@ -0,0 +1,73 @@
+---
+arch: x86_64
+disks:
```

Comment:
delete completely this section and replace by `num_disk` as the other keys are not used in the test.

Reply:
done.